src/services/catalog.service.ts

```diff
@@ -158,6 +160,7 @@ export class ImageItem implements IServiceItem {
     this.longDescription = this.getLongDescription();
     this.catsBySubCats = this.getCategoriesBySubCategories();
     this.isBuilder = _.includes(this.tags, 'builder');
```

Comment:
We need to check that there is a corresponding tag in `status.tags` that matches the builder spec tag.

Image streams have `spec.tags` that are the tag definitions and `status.tags` that are the tags which are currently loaded / exist.